modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -7,59 +7,29 @@
 
 You can establish a binding connection with Operator-backed components.
 
-This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
-The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+This procedure walks through an example of creating a binding connection between a PostgresSQL Database service and a Spring PetClinic sample application.
+
+To create a binding connection with a service that is backed by the Crunchy Postgres for Kubernetes, you must add the Crunchy Postgres for Kubernetes operator to the *OperatorHub*, and then install the Operator. The Crunchy Postgres for Kubernetes Operator provides PostgresCluster resource which then can be used to deploy a  PostgreSQL database, which exposes the binding information in secrets, config maps, status, and spec attributes.
+
+When you create the service binding, the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
```

Comment:
@Max-cx and @Dhruv-Soni11  It is crucial in this context to use the word project for consistency's sake because this is the technical term we are using right from the concept stage as preferred by the SMEs of SBO. I suggest and insist on retaining this word as is and refraining from using the suggestion `inject`.

Reply:
Thanks for providing this suggestion @Srivaralakshmi. I will keep the term as is as per your suggestion.

Reply:
- +1 what Max has expressed here ^^^.
- Our own official definition is here. "In OpenShift Online, **a project is a Kubernetes namespace with additional annotations**. Namespaces provide a unique scope for: Named resources to avoid basic naming collisions." https://docs.openshift.com/online/pro/architecture/core_concepts/projects_and_users.html
- Some folks are still using the Kubernetes' legacy terminology, where project == namespace. Get clarification from the SME which case we're talking about and apply the correct term according to our definition of the term.
- Avoid using "project" as a verb in this context. It creates too much semantic noise and confusion no matter the expertise of the reader. Look for a synonym such as "inject" or "insert".

Reply:
Would you consider using a helper verb: "can project"?

Side note: Usability studies show that most users follow a search result to a specific section of technical documentation. They rarely read the whole page from top to bottom. When confused, they are very likely to start a new web search than to dig deeper into the content.

Reply:
Thanks for the suggestion. Only in places where it suits. In this particular instance, I prefer to retain the usage as is.

Reply:
Thank you very much everyone for the detailed discussion about the usage of this term in this context. I am not going to change much and keep it as is in this instance. Fixed.